admin/app/controllers/authenticated/certifications/certification/informations.js

```diff
@@ -178,6 +183,26 @@ export default class CertificationInformationsController extends Controller {
     this.certification.birthdate = lastSelectedDateFormatted;
   }
 
+  @action
+  onCancelCourseButtonClick() {
+    const confirmMessage = 'Êtes vous sur de vouloir annuler cette certification ? Cliquer sur confirmer pour poursuivre';
+
+    this.confirmAction = 'onCancelCourseConfirmation';
+    this.confirmMessage = confirmMessage;
+    this.displayConfirm = true;
+  }
+
+  @action
+  async onCancelCourseConfirmation() {
+    await this.certification.save({
```

Comment:
Côté PixAdmin à ma connaissance on expérimente une dépendance appelée les memberActions. Ca permet de camoufler un peu cette mocheté de tricks qu'on fait avec les adapterOptions là.
En gros ça permet d'avoir un appel élégant sur le modèle du style : `await this.certification.cancel();`